config/scootconfig/scheduler_config_modules.go

```diff
@@ -0,0 +1,33 @@
+package scootconfig
```

Comment:
for consistency, can this file be named scheduler_modules.go? Looks like the other filenames don't mention config